test/test_transformers.py

```diff
@@ -1753,6 +1753,96 @@ def test_scaled_dot_product_fused_attention_vs_math_cpu(
             self.assertEqual(grad_k_actual, grad_k_ref, atol=atol, rtol=rtol)
             self.assertEqual(grad_v_actual, grad_v_ref, atol=atol, rtol=rtol)
 
+    @onlyCPU
+    @parametrize("fused_kernel", [SDPBackend.FLASH_ATTENTION])
+    @parametrize("dtype", [torch.float64, torch.float32, torch.bfloat16])
+    @parametrize("batch_size", [2, 12])
+    @parametrize("seq_len", [267, 1030])
```

Comment:
does this support different sequence lengths between q, and kv I think we should add a check for that

Reply:
Thanks. Removed `seq_len` and added two more params `q_seq_len` and `kv_seq_len`, to check different sequence lengths.